projects/canopy/src/lib/hero-img/hero-img-card/hero-img-card.component.scss

```diff
@@ -1,7 +1,7 @@
 @import '../../../styles/mixins';
 
 .lg-hero-img-card {
-  background-color: var(--hero-img-bg-color-rgb);
+  background-color: var(--hero-img-bg-color);
```

Comment:
The rgb colour should work but i think you need to write:
```
background-color: rgba(var(--hero-img-bg-color-rgb), 1)
```

Reply:
BTW you can test this in your project by building the app in the dotcom `package.json` you can add this script:
```
"util:copy-canopy": "rimraf node_modules/@legal-and-general/canopy && copyfiles -u 3 \"../canopy/dist/canopy/**/*\" <your-folder-path>dotcom/node_modules/@legal-and-general/",
```

Reply:
@elenagarrone thank you, that's a better solution :)

Reply:
Thanks @elenagarrone - we've also implemented a script that copied the local version of canopy to the dotcom project. But perhaps your solution is easier to work with